src/services/WizardService.ts

```diff
@@ -8,12 +9,18 @@ export interface IWizardState {
   registrationData: IRegistrationData;
 }
 
-export interface IWizardStage {
+export interface IWizardStage<Errors = any> {
```

Comment:
Could be a good practice to always use `unknown` over `any`.
This is what somehow stuck to my mind.

Quick search:
https://stackoverflow.com/questions/51439843/unknown-vs-any

Reply:
Thanks for the hint